src/callbacks.py

```diff
@@ -5,15 +5,14 @@
 from .losses.ctdet_loss import decode_centernet_predictions
 from .metrics import class_agnostic_mean_ap, calculate_map, construct_mAP_list_from_bboxes
 
-from catalyst.dl import utils
-
 
 def detach(tensor) -> np.ndarray:
```

Comment:
let's return `from catalyst.dl import utils` back and use `utils.detach` everywhere it needed

Reply:
done